src/widget/wtracktableview.cpp

```diff
@@ -1386,12 +1386,30 @@ void WTrackTableView::slotImportTrackMetadata() {
         return;
     }
 
+    for (const QModelIndex& index : indices) {
+        TrackPointer pTrack = trackModel->getTrack(index);
+        if (pTrack && pTrack->isDirty()) {
+            if (QMessageBox::Apply == QMessageBox::question(
+                    nullptr,
+                    tr("Import Track Metadata"),
+                    tr("One or more selected tracks have unsaved changes. Continue anyway?"),
```

Comment:
"unsaved changes" is a bit unclear. What kind of changes are you referring to? How about: "One or more selected tracks have metadata in Mixxx that has not been saved to file tags. Continue anyway?"

Reply:
Do we need this messages at all?

I think we need it only if export is enabled, and if the user has actually changed some metadata that is intended to be exported back to the file.
The dirty flag is not a good indicator for this. Because it tracks the data to be stored in the database like played counter as well.

It would be ok for me to omit the message box here, since a "working" solution requires a lot more changes, and the case of loosing data can be neglected, because this feature is used if you have edited the metadata in a third party tool and if one has edited Mixxx metadata before, it was most likely accidental, and the import is intended to fix this. So the data loss I expected and desired in this case anyway.

Reply:
Sounds reasonable to me.